lib/location/location_core.c

```diff
@@ -94,15 +85,17 @@ static const struct location_method_api method_gnss_api = {
 #endif
 };
 #endif
+
+#if defined(CONFIG_LOCATION_METHOD_CELLULAR) || defined(CONFIG_LOCATION_METHOD_WIFI)
```

Comment:
It's hard to see where this ends because there's so many #if's. Maybe you could move this after line 120, looks like it doesn't have to be here.

Reply:
Putting it back there :)